src/main/java/com/google/opensesame/servlets/UserObject.java

```diff
@@ -0,0 +1,123 @@
+package com.google.opensesame.servlets;
+
+import com.google.opensesame.github.GitHubGetter;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GitHub;
+
+public class UserObject {
+  private final ArrayList<Long> projectIDs;
+  private final ArrayList<String> menteeIDs;
+  private final ArrayList<String> interestTags;
+  private final String bio;
+  private final String email;
+  private final String gitHubID;
+  private final String gitHubURL;
+  private final String image;
+  private final String location;
+  private final String name;
+  private final String userID;
+  private final Boolean isMentor;
+
+  public UserObject(
+      String userID,
+      String gitHubID,
+      ArrayList<String> interestTags,
+      ArrayList<String> menteeIDs,
+      ArrayList<Long> projectIDs,
```

Comment:
Would it be possible to replace `PersonObject` and `MentorObject` with this class? If it would push back the timeline too much don't worry about it.